src/libstd/process.rs

```diff
@@ -1016,27 +1016,115 @@ impl fmt::Debug for Stdio {
 
 #[stable(feature = "stdio_from", since = "1.20.0")]
 impl From<ChildStdin> for Stdio {
+    /// Converts a `ChildStdin` into a `Stdio`
+    ///
+    /// # Examples
+    ///
+    /// `ChildStdin` will be converted to `Stdio` using `Stdio::from` under the hood.
+    ///
+    /// ```rust
+    /// use std::process::{Command, Stdio};
+    ///
+    /// let reverse = Command::new("rev")
+    ///     .stdin(Stdio::piped())
+    ///     .spawn()
+    ///     .expect("failed reverse command");
+    ///
+    /// let _echo = Command::new("echo")
+    ///     .arg("Hello, world!")
+    ///     .stdout(reverse.stdin.unwrap()) // Converted into a Stdio here
+    ///     .output()
```

Comment:
This one should use a `spawn()`, and on line 1030 should be an `output()`, which allows us to use `assert_eq!` as well.

Reply:
When I change `spawn()` to be an `output()` rustc complain with:
```rust
error[E0609]: no field `stdin` on type `std::process::Output`
  --> src/main.rs:12:25
   |
12 |         .stdout(reverse.stdin.unwrap()) // Converted into a Stdio here
   |                         ^^^^^ unknown field
   |
   = note: available fields are: `status`, `stdout`, `stderr`
```
[Playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2015&)gist=92f64ab621dd57dd2226cb6b80e01258

With `spawn()` however, I have not managed to get the output to `assert_eq!`.

Reply:
I see. Just squash the commits then ;)